src/panels/profile/ha-refresh-tokens-card.js

```diff
@@ -96,6 +96,9 @@ class HaRefreshTokens extends LocalizeMixin(EventsMixin(PolymerElement)) {
 
   async _handleDelete(ev) {
     const token = ev.model.item;
+    if (token.is_current) {
```

Comment:
Should we just not show the delete icon for current?

Reply:
A disabled button shouldn't fire the click event, have to check it, shouldn't be in this PR.